packages/camera/camera_avfoundation/ios/Classes/FLTCam.h

```diff
@@ -45,6 +47,19 @@ NS_ASSUME_NONNULL_BEGIN
                        orientation:(UIDeviceOrientation)orientation
                captureSessionQueue:(dispatch_queue_t)captureSessionQueue
                              error:(NSError **)error;
+
+- (instancetype)initWithCameraName:(NSString *)cameraName
```

Comment:
Needs a declaration comment.

Also, does this really need to be here instead of `FLTCam_Test.h`?

Reply:
fixed